assignments/stretch-function-conversion.js

```diff
@@ -1,23 +1,32 @@
 // Take the commented ES5 syntax and convert it to ES6 arrow Syntax
 
 // let myFunction = function () {};
+let myFunction = () => {};
 
 // let anotherFunction = function (param) {
 //   return param;
 // };
 
+let anotherFunction = param => {return param};
+
```

Comment:
```suggestion
let anotherFunction = param => param;

```
You can leave the braces off here and it'll still return the value of param